submissions/IhorOzerov/html-css-popup/styles.css

```diff
@@ -0,0 +1,106 @@
+.menu-a:focus, a:focus, .greatest:focus-within {
+    outline: 1px solid rgb(74, 74, 254)
+}
+
+.visually-hidden {
+     position: absolute;
+     width: 1px;
+     height: 1px;
+     margin: -1px;
+     border: 0;
+     padding: 0;
+     clip: rect(0 0 0 0);
+     overflow: hidden;
+}
+
+.header-icons {
+     display: flex;
+     justify-content: flex-start;
+     align-items: center;
+     padding-left: 0;
+     margin: 0 0;
+}
+
+.h-image a{
+    text-decoration: none;
+    color: rgb(72, 94, 71);
+}
+
+label.more-btn:hover, .h-image:hover{
+    cursor: pointer;
+}
+
+li {
+     list-style: none;
+}
+
+.h-image {
+     width: 30px;
+     margin: 15px;
+}
+
+.visible-img {
+     width:55px;
+     height: 50px;
+}
+
+.menu {
+     width: 300px;
+     height: 450px;
+     background-color: rgb(226, 220, 220);
+     justify-items: center;
+     padding: 15px;
+     display: none;
+     grid-template-columns: repeat(3, 1fr);
+     grid-template-rows: 1fr 1fr 1fr 1fr 30px;
+     overflow-x: hidden;
+    box-sizing: border-box;
+    flex-wrap: wrap;
+    position: absolute;
+}
+
+#sq:checked ~ .menu{
```

Comment:
Try to keep your code style: use CSS classes for styles, IDs for js logic, or link inputs with labels. Avoid styling by HTML tags(unless it is some base style or reset) or id.